CMakeLists.txt

```diff
@@ -46,9 +46,93 @@ option(FF_USE_ALL_PREBUILT_LIBRARIES "Enable use of all pre-compiled libraries,
 # option for using Python
 option(FF_USE_GASNET "Run FlexFlow with GASNet" OFF)
 set(FF_GASNET_CONDUITS aries udp mpi ibv ucx)
-set(FF_GASNET_CONDUIT "mpi" CACHE STRING "Select GASNet conduit ${FF_GASNET_CONDUITS}")
+set(FF_GASNET_CONDUIT ${FF_GASNET_CONDUIT} CACHE STRING "Select GASNet conduit ${FF_GASNET_CONDUITS}")
```

Comment:
why do we need this change?

Reply:
I supposed we need to pass the variable to here. It is not necessary and I have reverted it.